frontend/src/routes/NewRequest.tsx

```diff
@@ -709,6 +721,20 @@ const Card = (props: CardProps) => {
               </button>
             </div>
           )}
+          {props.connectionType === "DATASOURCE" && (
```

Comment:
I think I'd like to enable this on a per connection basis in the settings (e.g. dump-enabled).
And also not show it completely for ms sql, since we don't support the dumping there.

Reply:
Right! I've adjusted it based on that logic, thanks!!